pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -506,6 +507,10 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(uid kubetypes.UID, n
 	// TODO: optimization: set maximum number of containers per container name to examine.
 	for i, c := range containers {
 		status, err := m.runtimeService.ContainerStatus(c.Id)
+		if crierror.IsNotFound(err) {
+			klog.V(5).InfoS("Container not found", "containerID", c.Id, "err", err)
```

Comment:
Why is this level 5？ Can it be consistent with the following log level?

Reply:
I think that not-found is an error we can tolerate, so I let it's level more lower

Reply:
I don't know that we need to log at this level here.  If the container isn't found, that's normal.